modules/openapi-generator/src/main/resources/dart2/class.mustache

```diff
@@ -107,6 +107,10 @@ class {{classname}} {
     {{/complexType}}
   {{/isDate}}
   {{/isDateTime}}
+    {{#required}}
+    if ({{name}} == null)
```

Comment:
Can you please also add the `{{^isNullable}} .. {{/isNulalble}}` check inside the `required`mustache tags?